frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-modal-enhanced.tsx

```diff
@@ -80,6 +82,17 @@ const VMWizardStorageModal: React.FC<VMWizardStorageModalProps> = (props) => {
     },
   ];
 
+  const getAllowedBusses = (): Set<string> => {
```

Comment:
should not be needed, there should be just one templateValidation at this point.